package.json

```diff
@@ -36,15 +37,15 @@
     "karma-typescript-preprocessor": "0.3.1",
     "lodash": "4.17.4",
     "powerbi-visuals-tools": "1.6.4",
-    "powerbi-visuals-utils-chartutils": "0.3.0",
-    "powerbi-visuals-utils-colorutils": "0.2.1",
-    "powerbi-visuals-utils-dataviewutils": "1.0.0",
-    "powerbi-visuals-utils-formattingutils": "1.0.0",
-    "powerbi-visuals-utils-interactivityutils": "0.3.1",
-    "powerbi-visuals-utils-svgutils": "1.0.0",
-    "powerbi-visuals-utils-testutils": "1.0.0",
-    "powerbi-visuals-utils-tooltiputils": "0.3.0",
-    "powerbi-visuals-utils-typeutils": "1.0.0",
+    "powerbi-visuals-utils-chartutils": "^1.0.0",
+    "powerbi-visuals-utils-colorutils": "^1.0.0",
+    "powerbi-visuals-utils-dataviewutils": "^1.0.0",
```

Comment:
please use 1.1.0, as far as i remember it had it's dependency fixed in that version